### PR TITLE
Stocks Station v1

### PR DESCRIPTION
**Pull request by JuliCarracedo**
This PR by @JuliCarracedo introduces the full implementation for Stock Station:

- SPA built on React components and the Redux store
- Design answers to the project's requirements
- Home page shows a set of companies that can be filtered by any data available in the API
- Those company cards show data according to the chosen filter
- Clicking a card will send the user to a dedicated page that shows more data for the selected company
- Live demo available: https://stocks-station.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,35 @@
-import logo from './logo.svg';
+/* eslint-disable no-nested-ternary */
```
Comment (arslanbisharat) [inline src/App.js]: - [x] Let's not disable linter rules. These are here to write clean code.

File: src/components/CompanyPage/CompanyPage.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-nested-ternary */
```
Comment (arslanbisharat) [inline src/components/CompanyPage/CompanyPage.js]: - [x] Let's not disable linter rules. These are here to write clean code.

File: src/test/stocksReducer.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-expressions */
```
Comment (arslanbisharat) [inline src/test/stocksReducer.test.js]: - [x] Let's not disable linter rules. These are here to write clean code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure your Pull Request is from Feature to Development following correct Gitflow. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 4]
```diff
@@ -1,70 +1,84 @@
...
+    - Git & GitHub
+    - FinancialModelingPrep API [(link here)](https://site.financialmodelingprep.com/developer/docs);
+    - NPM
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[Live Demo Link](stocks-station.netlify.app)
```
Comment (enaburekhan) [inline README.md]: **Optional**

- [ ] I get a 404 error when I click on the demo link in the readme. 

image.
I will suggest you edit the current link to include the https like so:  https://stocks-station.netlify.app/

**Review by enaburekhan — APPROVED** [review]
Hi JuliCarracedo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuliCarracedo** [comment]
MErge Stocks v1

**Pull request merged**